### PR TITLE
Remove passing storage to unneeded places

### PR DESCRIPTION
**Pull request by Tishka17**

### CODE REVIEW

File: annet/gen.py [hunk 1]
```diff
@@ -808,23 +802,16 @@ def devices(self) -> List[Device]:
...
         return []
 
-    def resolve_devices(self, storage: Storage, device_ids: Iterable[int]) -> List[Device]:
+    def resolve_devices(self, device_ids: Iterable[int]) -> List[Device]:
         devices = []
         for device_id in device_ids:
             device = self._devices_map[device_id]
-
-            # can not use self._storage here, we can be in another process
-            device.storage = storage
```
Comment (Tishka17) [inline annet/gen.py:817]: revert

File: annet/parallel.py [hunk 2]
```diff
@@ -131,36 +131,20 @@ def _pool_worker(pool, index, task_queue, done_queue):
                 if device_id:
                     span.set_attribute("device.id", device_id)
 
-                name = "invoke"
-                invoke_span_ctx = tracing_connector.get().start_as_linked_span(name, tracer_name=__name__)
-                capture_output_ctx = capture_output(cap_stdout, cap_stderr)
-
-                with invoke_span_ctx as invoke_span, capture_output_ctx as _:
-                    invoke_trace_id = invoke_span.get_span_context().trace_id
-                    if invoke_trace_id:
```
Comment (Tishka17) [inline annet/parallel.py:140]: TODO: add callback for tracing

File: annet/api/__init__.py [hunk 3]
```diff
@@ -185,28 +186,28 @@ def _print_pre_as_diff(pre, show_rules, indent, file=None, _level=0):
...
-        message = task_result.extra["message"]
-    else:
-        status = "OK" if task_result.exc is None else "FAIL"
-        status_color = colorama.Fore.GREEN if status == "OK" else colorama.Fore.RED
-        message = "" if status == "OK" else str(task_result.exc)
-    progress_logger.info(message,
-                         perc=perc, fqdn=fqdn, status=status, status_color=status_color,
-                         worker=task_result.worker_name, task_time=elapsed_time)
-    return task_result
+class PoolProgressLogger:
```
Comment (Tishka17) [inline annet/api/__init__.py:189]: breaks compatibility

**Review by azryve — COMMENTED** [review]
The same stuff should be tested here, also there is a removed tracer which should not been here in the first place in _pool_worker 😭 @drtyrsa @grigory51 

Otherwise - no problems noted

**Review by azryve — APPROVED** [review]
Still no complaints, everything looks pretty much straightforward

**Comment by azryve** [comment]
This PR has too many things related to #18, trying to separate it progressively more confusing, so I desided to join them and test internally in tandem

**Pull request closed without merging**